infomaniak-build-tools/linux/build-ci-amd64.sh

```diff
@@ -66,6 +64,15 @@ cmake -B$BUILDDIR -H$BASEPATH \
     "${CMAKE_PARAMS[@]}" \
 
 make -j4
+
+objcopy --only-keep-debug ./bin/kDrive $CONTENTDIR/kDrive-amd64.dbg
+objcopy --strip-debug ./bin/kDrive
+objcopy --add-gnu-debuglink=$CONTENTDIR/kDrive-amd64.dbg ./bin/kDrive
+
```

Comment:
```suggestion
# Client
```

Reply:
Suggestion: create a function and call it twice with different arguments.

Reply:
Created the function